controllers/blogController.js

```diff
@@ -0,0 +1,88 @@
+var models = require('../models');
+
+var blog=function(){
+
+};
+
+blog.getSinglepost = function(req,res)
+{
+  var id=req.params.id;
+  models.blogs.getSinglepost(models,id,function(error,response)
+  {
+
+    if(error)
+    {
+      return res.status(404).send(response);
+    }
+    return res.status(200).send(response);
+  });
+};
+
+
+blog.getAllblog = function(req,res)
+{
+
+  models.blogs.getAllblog(models,req,function(error,response)
+  {
+
+    if(error)
+    {
+      return res.status(404).send(response);
```

Comment:
status 404 is not found. please check which status code to be used if there are errors